### PR TITLE
chore: clean up mutator tests

### PR DESCRIPTION
**Pull request by BeksOmega**
## The basics

- [X] I [validated my changes](https://developers.google.com/blockly/guides/contribute/core#making_and_verifying_a_change)

### Resolves

Fixes N/A

Follow on to #7423 that fixes up the mutator tests.

### Proposed Changes

* Reformats the mutator tests to conform to the style guide.
* Changes the `connect` method to not need to take in the element selector.
* Adds some helpers to remove more concrete selectors from the test.

### Reason for Changes

All of these things are to aid in readability.

Changes to the `connect` method also involved partially fixing https://github.com/google/blockly/issues/7432, so we may want to delay those changes for now since they're not complete.

### Test Coverage

This is tests!

### Documentation

N/A

### Additional Information

N/A
](https://github.com/google/blockly/pull/7393/files)

### CODE REVIEW

**Review by NeilFraser — APPROVED** [review]

File: tests/browser/test/mutator_test.js [hunk 1]
```diff
@@ -4,94 +4,77 @@
...
   this.timeout(0);
 
-  // Setup Selenium for all of the tests
   suiteSetup(async function () {
     this.browser = await testSetup(testFileLocations.PLAYGROUND);
   });
 
-  test('This test mutating a block creates more inputs', async function () {
-    await testingMutator(this.browser, screenDirection.LTR);
+  test.only('Mutating a block creates more inputs', async function () {
```
Comment (BeksOmega) [inline tests/browser/test/mutator_test.js:27]: Remove `only`.

File: tests/browser/test/test_setup.js [hunk 2]
```diff
@@ -306,21 +307,20 @@ async function connect(
...
       draggedConnection,
       mutatorBlockId,
     );
     targetLocation = await getLocationOfBlockConnection(
       browser,
-      targetBlock,
+      targetBlock.id,
       targetConnection,
       mutatorBlockId,
     );
```
Comment (BeksOmega) [inline tests/browser/test/test_setup.js:326]: I think we can remove this else and just always pass `mutatorBlockId` (which may be undefined).

File: core/flyout_base.ts [hunk 3]
```diff
@@ -1214,6 +1214,8 @@ export abstract class Flyout extends DeleteArea implements IFlyout {
 
     // Clone the block.
     const json = blocks.save(oldBlock) as blocks.State;
+    // TODO: Add a saveIds parameter to `save`.
+    json['id'] = undefined;
```
Comment (BeksOmega) [inline core/flyout_base.ts:1218]: I'm going to revert this change because I don't want to half-fix the issue.
